test/source/tests/decrypt.ts

```diff
@@ -610,15 +654,19 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithBrowser: Te
         t.mockApi!.configProvider = new ConfigurationProvider({
           attester: singlePubKeyAttesterConfig(acctEmail, somePubkey),
         });
-        await BrowserRecipe.setUpCommonAcct(t, browser, 'compatibility');
-        await BrowserRecipe.pgpBlockVerifyDecryptedContent(t, browser, {
-          content: ['This will will have a larger attachment below', 'image-large.jpg'],
-          encryption: 'encrypted',
-          signature: 'not signed',
-          params:
-            '?account_email=flowcrypt.compatibility%40gmail.com&frame_id=frame_yVMKFLRDiY&message=&message_id=1622eaa286f90737&senderEmail=&is_outgoing=___cu_false___',
-          expectPercentageProgress: true,
-        });
+        const { authHdr } = await BrowserRecipe.setUpCommonAcct(t, browser, 'compatibility');
+        await BrowserRecipe.pgpBlockVerifyDecryptedContent(
+          t,
+          browser,
+          '1622eaa286f90737',
+          {
+            content: ['This will will have a larger attachment below', 'image-large.jpg'],
+            encryption: 'encrypted',
+            signature: 'not signed',
+            expectPercentageProgress: true,
```

Comment:
As decryption now happens in the content script context, the ajax request to download the large attachment is handed over to the background page as per #1597
Thus displaying progress will be more complicated. The background page may send progress callbacks in the form of browser messages. They may be handled by the pgp_block frame directly.
That is, the content script creates a pgp_block iframe (already implemented),
initiates data download in background and receives data (already implemented),
meanwhile the pgp block receives messages from the background to display progress updates. The content script should pass additional data to the background, like frameId.
Not too complicated, @tomholub ?

Reply:
i think it is what it is, seems ok